scripts/question_answering/README.md

```diff
@@ -70,6 +70,13 @@ python run_squad.py \
     --overwrite_cache \
 ```
 
+We support multi-GPU training via horovod:
```

Comment:
Would you like to include performance numbers here, pointing out how much speedup users can expect by using horovod?

Reply:
Sure, the  performance numbers may quite different depend on which model we are training.  I only included the numbers of roberta large here.

Reply:
Thank you for adding it. It will still provide a rough estimate to users